media/css/firefox/family/index.less

```diff
@@ -253,8 +253,8 @@ html[dir="rtl"] #header h1 {
 /*--------------------------------------------------------------------------*/
 // Base styles for product groups
 .product-list {
-    .clearfix();
-    .flexbox();
+    .clearfix;
+    .flexbox;
```

Comment:
This also creates problems in IE9-- :(

![windows 7 - ie9](https://cloud.githubusercontent.com/assets/10350960/11004416/fc3c64e4-846c-11e5-970e-f4197dbcc878.png)